mantidproject/mantid
# Update Windows buildscripts to MSVS 2017

**Description of work.**

Switch Windows builds MSVS 2017.

**Report to:** [nobody]. 

**To test:**

The builds should pass. Check the build logs to see that it uses Visual Studio 2017. It's probably worth building a package and checking it installs and starts okay.

Note that I have tested a clean build with a [separate temporary job on Jenkins](http://builds.mantidproject.org/job/test-win7-2017/9/).

*There is no associated issue.*

*This does not require release notes* because **it is an internal developer change.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
